reference/7.4/PSReadLine/Get-PSReadLineOption.md

```diff
@@ -44,35 +44,43 @@ HistorySaveStyle                       : SaveIncrementally
 HistorySearchCaseSensitive             : False
 HistorySearchCursorMovesToEnd          : False
 MaximumHistoryCount                    : 4096
-ContinuationPrompt                     : >>
+ContinuationPrompt                     :
 ExtraPromptLineCount                   : 0
-PromptText                             : {> }
+PromptText                             :
 BellStyle                              : Audible
 DingDuration                           : 50
 DingTone                               : 1221
-CommandsToValidateScriptBlockArguments : {ForEach-Object, %, Invoke-Command, icm...}
+CommandsToValidateScriptBlockArguments : {ForEach-Object, %, Invoke-Command, icm…}
 CommandValidationHandler               :
 CompletionQueryItems                   : 100
 MaximumKillRingCount                   : 10
 ShowToolTips                           : True
 ViModeIndicator                        : None
-WordDelimiters                         : ;:,.[]{}()/\|^&*-=+'"---
+WordDelimiters                         : ;:,.[]{}()/\|!?^&*-=+'"–—―
 AnsiEscapeTimeout                      : 100
+PredictionSource                       : HistoryAndPlugin
+PredictionViewStyle                    : InlineView
+TerminateOrphanedConsoleApps           : False
 CommandColor                           : "`e[93m"
 CommentColor                           : "`e[32m"
-ContinuationPromptColor                : "`e[97m"
-DefaultTokenColor                      : "`e[97m"
+ContinuationPromptColor                : "`e[37m"
+DefaultTokenColor                      : "`e[37m"
 EmphasisColor                          : "`e[96m"
 ErrorColor                             : "`e[91m"
+InlinePredictionColor                  : "`e[100m"
 KeywordColor                           : "`e[92m"
-MemberColor                            : "`e[97m"
+ListPredictionColor                    : "`e[33m"
+ListPredictionSelectedColor            : "`e[48;5;238m"
+ListPredictionTooltipColor             : "`e[97;2;3m"
+MemberColor                            : "`e[37m"
 NumberColor                            : "`e[97m"
-OperatorColor                          : "`e[90m"
-ParameterColor                         : "`e[90m"
-SelectionColor                         : "`e[30;107m"
+OperatorColor                          : "`e[95m"
+ParameterColor                         : "`e[95m"
+SelectionColor                         : "`e[92m`e[100m"
```

Comment:
The default values shown for me is:
```
OperatorColor                          : "`e[90m"
ParameterColor                         : "`e[90m"
SelectionColor                         : "`e[30;47m"
```

Reply:
My bad for not running this under `-noprofile`.